internal/dcache/debug/callbacks.go

```diff
@@ -90,6 +93,27 @@ func readStatsCallback(pFile *procFile) error {
 	return nil
 }
 
+// proc file: logs
+// On first open, it triggers collection of logs from all nodes via RPC and returns a JSON summary.
+// Tarballs are stored on disk. This file only returns metadata mapping of node IDs to log tarball paths.
+//
+// By default, we use the default work dir for storing the collected logs and
+// we also collect only the most recent log file from each node by default.
+//
+// If users want to specify a different directory or number of logs, they can use the
+// fs=debug/logs file with a write call to specify those parameters.
```

Comment:
the idea is not to allow collecting logs using write (and read).
write should purely only support configuring. Once onfigured it should remember the settings and subsequent reads of the logs file should honor those while collecting logs. So note that logs are only ever collected by reading fs=debug/logs file, never by writing to it.

Reply:
Updated